README.md

```diff
@@ -1,3 +1,43 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+#README
+
+	###Check-Ins
+
+  We had async check-ins in the morning including status updates, with a requested response by 9 AM MT. If working in the evening, Slack status updates before signing off for the next morning are acceptable.
+Live group check-ins during class group time. Stand up responses state the status of assigned tasks; what the programmer will be working on that day; and questions, comments and concerns that inform the group.
+
+  ###Workflow
+
+Team members assign tasks to themselves, changing status to “In Progress” and creating a pull request when complete, changing status again to “Pull Request”. Pull request workflow is submitted when ready to merge or if stuck or need help, followed by a slack to the group detailing and documenting further any issues. A team member will respond to claim ownership of the review, and reviewers are designated by morning and evening, with Garrett and Ashley reviewing in the AM and Matthew and Lauren taking the PM review shifts. The timeframe for reviewing is up to 12 hours after pull request submission.  Project Board:  https://github.com/users/aplasket/projects/4
+
+###Project Organization
+
+For Project Managers we chose Github Projects as we found it was potentially more versatile than other manager programs and directly integrated with GitHub. We researched Trello as well but found that some of its features required purchase for full usability and GitHub projects offered better ease of communication between team members. Ashley was chosen to host the repo for this project. Retro was planned for 04/25/2023 at 9 AM MT.
+
+  ###Code Design
+
+We established shared conventions for VS Code. We would use double quotes (“”). We would have a vertical attr_reader. Hashes <> Array is expected versus .to eq(hash).
+
+  ###DTR Document
+https://docs.google.com/document/d/1aQTaoqGz3xtM8JAw5DZGTMAQPFfMce8zIxMc6ZQXAz8/edit?usp=sharing
+Initial Document created 04/23/2023
```

Comment:
The document was created on 04/17/2023. Can you please update this date?

Reply:
Updated date to 04/17/2023